src/commodore/index.ts

```diff
@@ -38,7 +38,7 @@ export const defaultExtraConfig = {
   distributionRegex:
     /^distribution\/(?<distribution>[^\/]+)(?:\/cloud\/(?<cloud>.+)\.ya?ml|(?:\/.+)?\.ya?ml)$/,
   cloudRegionRegex:
-    /^cloud\/(?<cloud>[^\/]+)(?:\/(?<region>.+)\.ya?ml|\.ya?ml)$/,
+    /^cloud\/(?<cloud>[^\/]+)(?:\/(?<region>[^\/]+)\.ya?ml|[^\/]*\.ya?ml)$/,
```

Comment:
I think the actual new regex should be

```suggestion
    /^cloud\/(?<cloud>[^\/]+)(?:\/(?:(?<region>[^\/]+)|.+)\.ya?ml|\.ya?ml)$/,
```

the last `\.ya?ml` was originally only supposed to match files like `cloud/<cloudname>.yaml`, but with the restricted region regex we need to add an additional option to match everything which has prefix `cloud/<cloudname>/` but doesn't match `cloud/<cloudname>/<name>.ya?ml` in the second top-level group.